CHANGES.rst

```diff
@@ -8,11 +8,16 @@ New Features
 
 - ``astropy.constants``
 
-  - Constants are now organized into version classes, with updated physical
-    constants in the CODATA2014 class and astronomical constants in the
-    ASTRON_V2_0 class. These updated constants are available in the
-    ``astropy.constants`` namespace. The constants for astropy v1.3
-    are retained in separate version classes. [#4958, #4109, #4397]
+  - Constants are now organized into version modules, with updated physical
+    constants in the CODATA2014 modules and astronomical constants in the
+    IAU2015 modules. These updated constants are available in the
+    ``astropy.constants`` namespace, as well as the ASTROPYCONST20 module.
+    The constants for astropy v1.3 are retained in separate version modules
+    CODATA2010, IAU2012 and ASTROPYCONST13. [#4958, #6083]
+
+  - The GM mass parameters recommended by IAU 2015 Resolution B 3 have been
+    added as ``GM_sun``, ``GM_jup``, and ``GM_earth``, for the Sun,
+    Jupiter and the Earth. [#4153]
```

Comment:
nitpicking yet again, but could you list the PR number rather than (or in addition to) the issue number it implements?